codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
```

Comment:
I would prefer an `--omit=` option in run-tests.sh script, so it isn't tied to specific service processing the coverage data, and will work when inspecting the data locally too.